app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java

```diff
@@ -72,6 +96,11 @@ public View getChildView(int groupPosition, final int childPosition,
         return convertView;
     }
 
+    /**
+     * returns total children count.
```

Comment:
Be consistent. If you start with capital letters make sure you start with Capital in each comment